exercises/armstrong-numbers/example.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+  validate: function(input) {
+    var digits = (input + '').split('');
+    var sum = digits.reduce(function (total, current) {
+      return total += Math.pow(current, digits.length);
```

Comment:
do not the assign during return
`total += Math.pow(current, digits.length);`
 `return total`